src/internal-packages.lisp

```diff
@@ -203,6 +203,7 @@
    :*window-scroll-functions*
    :*window-size-change-functions*
    :*window-show-buffer-functions*
+   :*last-focused-window*
```

Comment:
Maybe this variable should be encapsulated, what do you think?

Reply:
Or if you need to reference it from the outside package, maybe use the `(last-focusd-window)` function?

Reply:
Yes I think you are right. I'll export a `(last-focused-window)` function instead. It is only `switch-to-last-focused-window` which needs to access externally but I think it is best to keep the logic with the command.